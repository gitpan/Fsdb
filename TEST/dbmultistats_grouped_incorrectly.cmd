prog='dbmultistats'
args='-S -S -k experiment duration'
in=TEST/dbmapreduce_grouped_incorrectly.in
cmp='diff -cb '
portable=false
subprogs=dbstats
