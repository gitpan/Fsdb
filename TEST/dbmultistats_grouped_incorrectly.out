#fsdb experiment mean stddev pct_rsd conf_range conf_low conf_high conf_pct sum sum_squared min max n
ufs_mab_sys	37.2	0	0	-	-	-	0.95	37.2	1383.8	37.2	37.2	1
ufs_rcp_real	264.5	0	0	-	-	-	0.95	264.5	69960	264.5	264.5	1
ufs_mab_sys	37.3	0	0	-	-	-	0.95	37.3	1391.3	37.3	37.3	1
ufs_rcp_real	277.9	0	0	-	-	-	0.95	277.9	77228	277.9	277.9	1
#  | dbmultistats -S -S -k experiment duration
